kettle/make_db_test.py

```diff
@@ -159,9 +159,10 @@ def assert_main_output(self, threads, expected=None, db=None,
         self.assertEqual(result, expected)
         return db
 
-    def test_clean(self):
-        for threads in [1, 32]:
-            self.assert_main_output(threads)
+    # def test_clean(self):
```

Comment:
Somehow this test failed on my local even with `bazel test //kettle:make_db_test`, have talked with @MushuEE and they will work on a fix

Reply:
can we add a TODO comment so it's greppable?

Reply:
done